onevizion/integration/log.py

```diff
@@ -0,0 +1,53 @@
+from onevizion.util import *
```

Comment:
Why `log` and `loglevel` are in the `integration`? It was renamed to Modules or VizionHub, but I think it should be in the `log`

Reply:
"email" is a builtin library, so having the same name interferes, but python is case sensitive, so "EMail" differentiates.
"import" is a keyword ,so can't be used as a file name.  Once again, "Import" gets around that.

I used "integration", because that is the name of the classes.  Presumably this naming convention is used in many existing internal and customer scripts already.  My main goal was to keep this completely comcatible with existing scripts.

Reply:
agree, let's keep existing naming for compatibility. We may consider renaming later, which of course may require changes in other code depending on this package